
def error(*msg):
    print(*msg)
    exit(-1)
